### PR TITLE
Add warning for users of Google reCAPTCHA version 3 with low score

### PR DESCRIPTION
**Pull request by fredden**
## Purpose of this pull request

This pull request adds a warning regarding the implementation of Google reCAPTCHA within Magento, specifically regarding how version 3 has been implemented to completely block genuine users with a low score.

## Magento release version

This feature is present in Magento v2.4.

## Product editions

This change affects Magento Open Source and Adobe Commerce.

## Affected documentation pages

- https://docs.magento.com/user-guide/configuration/security/google-recaptcha-admin.html
- https://docs.magento.com/user-guide/configuration/security/google-recaptcha-storefront.html

## Additional information

I have not changed the 'recommended' labels in these documents although I do disagree with their current placement. I do not think that the current implementation of Google reCAPTCHA v3 in Magento is suitable for an ecommerce website.

### CODE REVIEW

File: src/configuration/security/google-recaptcha-admin.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ Before Google reCAPTCHA can be configured, you must ensure that your `PHP.ini`
 {: .bs-callout-info}
 The reCAPTCHA type you choose must match the type that is associated with the API key from your Google reCAPTCHA account.
 
+{:.bs-callout-warning}
+When using version 3, a genuine user with low score will be unable to proceed; when using version 2, a genuine user with a low score will receive an challenge. Consider carefully if genuine users with a low score should be given an opportunity to solve a challenge (version 2) or completely blocked (version 3).
```
Comment (leslietilling) [inline src/configuration/security/google-recaptcha-admin.md:64]:
```suggestion
When using reCAPTCHA version 3, a genuine user with low score cannot proceed. For version 2, a genuine user with a low score receives a challenge. Consider carefully if genuine users with a low score should have an opportunity to solve a challenge (version 2) or be completely blocked (version 3).
```

File: src/configuration/security/google-recaptcha-storefront.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ Before Google reCAPTCHA can be configured, you must ensure that your `PHP.ini`
 {: .bs-callout-info}
 The reCAPTCHA type you choose must match the type that is associated with the API key from your Google reCAPTCHA account.
 
+{:.bs-callout-warning}
+When using version 3, a genuine user with low score will be unable to proceed; when using version 2, a genuine user with a low score will receive an challenge. Consider carefully if genuine users with a low score should be given an opportunity to solve a challenge (version 2) or completely blocked (version 3).
```
Comment (leslietilling) [inline src/configuration/security/google-recaptcha-storefront.md:62]:
```suggestion
When using reCAPTCHA version 3, a genuine user with low score cannot proceed. For version 2, a genuine user with a low score receives a challenge. Consider carefully if genuine users with a low score should have an opportunity to solve a challenge (version 2) or be completely blocked (version 3).
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
This is good information for merchants to know. Just some suggested edits for style and clarity.

**Comment by fredden** [comment]
Thanks for the suggestions.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**